spec/fig_offchain_prot.tex

```diff
@@ -73,29 +74,32 @@
 								$\hatmT \gets \hatmT \cup \{\tx\}$ \;
 								% issue snapshot if we are leader
 								\If{$\hats = \bar{\mc S}.s \land \hpLdr(\bar{\mc S}.s + 1) = i$}{
-									\Multi{} $(\hpRS,\hatv,\bar{\mc S}.s+1, \hatmT, \tx_{\omega})$ \;
+									\Multi{} $(\hpRS,\hatv,\bar{\mc S}.s+1,\hatmT,\tx_\omega\red{,\tx_\alpha})$ \;
 								}
 							}
 						}
 
 						\vspace{12pt}
 
 						%%% REQ SN
-						\On{$(\hpRS,v,s,\mT_{\mathsf{req}}, \tx_{\omega})$ from $\party_j$}{
+						\On{$(\hpRS,v,s,\mT_{\mathsf{req}}, \tx_\omega \red{, \tx_\alpha})$ from $\party_j$}{
```

Comment:
Must: fix inconsistencies in `reqSn` argument sequence

Sometimes we use $tx_\alpha$ sometimes $tx_\omega$ first.